recipes/jerryscript/all/conanfile.py

```diff
@@ -31,17 +98,59 @@ def _source_subfolder(self):
     @property
     def _build_subfolder(self):
         return "build_subfolder"
+    
+    def package_id(self):
+        if not self.info.options.profile in self._predefined_profiles:
+            with open(self.info.options.profile, "r") as profile_file:
+                self.info.options.profile = profile_file.read()
```

Comment:
```suggestion
        if not self.options.profile in self._predefined_profiles:
            self.info.options.profile = tools.load(str(self.options.profile))
```

Reply:
The suggestion was not good, see https://github.com/AGreat1/conan-center-index/pull/1